src/App.tsx

```diff
@@ -39,6 +43,9 @@ function App() {
         <NewTaskButton />
       </form>
       <TasksList tasks={tasks} onComplete={handleComplete} />
+      {tasks.length > 0 && (
+        <IncompleteTasksCounter counterLabel={counterLabel} />
+      )}
```

Comment:
в целом все хорошо, но я бы все таки передал в компонент count, а функцию лэйбла применил внутри компонента. От части это вкусовщина. Но в данном случае это в целом просто чтобы оправдать создание отдельного компонента.

У меня на работе это скорее всего было бы вообще так
```
      {tasks.length > 0 && (
        <Text (а тут пропсы, которые определяют стили текста доступные в приложении)>{counterLabel}</Text>
      )}
```

```suggestion
      {tasks.length > 0 && (
        <IncompleteTasksCounter count={getIncompleteTasksCount({ tasks })} />
      )}
```